app/models/vm_or_template/operations/relocation.rb

```diff
@@ -1,4 +1,16 @@
 module VmOrTemplate::Operations::Relocation
+  extend ActiveSupport::Concern
+
+  included do
+    supports_not :live_migrate
+    supports_not :warm_migrate
+    supports :migrate do
+      if blank? || orphaned? || archived?
+        unsupported_reason_add(:migrate, "Migrate operation in not supported.")
```

Comment:
Does this need translation around the `"Migrate operation in not supported."`?

Reply:
addressed